vnext/Microsoft.ReactNative/Modules/PaperUIManagerModule.cpp

```diff
@@ -126,20 +126,21 @@ class UIManagerModule : public std::enable_shared_from_this<UIManagerModule>, pu
 
   void createView(int64_t reactTag, std::string viewName, int64_t rootTag, React::JSValueObject &&props) noexcept {
     m_nativeUIManager->ensureInBatch();
-    auto viewManager = GetViewManager(viewName);
-    auto node = viewManager->createShadow();
-    node->m_className = std::move(viewName);
-    node->m_tag = reactTag;
-    node->m_rootTag = rootTag;
-    node->m_viewManager = viewManager;
+    if (auto viewManager = GetViewManager(viewName)) {
+      auto node = viewManager->createShadow();
+      node->m_className = std::move(viewName);
+      node->m_tag = reactTag;
+      node->m_rootTag = rootTag;
+      node->m_viewManager = viewManager;
 
-    node->createView(props);
+      node->createView(props);
 
-    m_nativeUIManager->CreateView(*node, props);
+      m_nativeUIManager->CreateView(*node, props);
 
-    m_nodeRegistry.addNode(shadow_ptr(node), reactTag);
+      m_nodeRegistry.addNode(shadow_ptr(node), reactTag);
 
-    node->updateProperties(props);
+      node->updateProperties(props);
+    }
```

Comment:
else, assert / failfast?

Reply:
I thought a no-op would be acceptable. It doesn't seem like CodeQL accepts asserts as valid checks, given the [CHAKRA_ASSERTDO in ChakraPlatform.cpp](https://github.com/microsoft/react-native-windows/pull/10007/files#diff-4c3ed2ceddc968533cf93b057e9939a474dec718a8910e35532c6f3cba249d9bR31) above.